app/src/main/java/org/mozilla/fenix/migration/MigrationProgressActivity.kt

```diff
@@ -95,12 +95,12 @@ internal val whiteList = mapOf(
     Settings to R.string.settings_title
 )
 
-internal fun MigrationResults.toItemList() = filterKeys {
+internal fun MigrationResults.toItemList(migrationCompleted: Boolean = false) = filterKeys {
     whiteList.keys.contains(it)
 }.map { (type, status) ->
     MigrationItem(
         type,
-        status.success
+        if (migrationCompleted) status.success else false
```

Comment:
I don't think this is the intention here. We DO want to show the state of every item as it is completed. However right now we only show any items at all if they are completed. Because the state from the store does only include completed items. Since we already have a whitelist here we could just use this to "fill up" the list for all items that are not completed. Or we add the state for incompleted migration items from AC.